src/data/familiars.txt

```diff
@@ -6,6 +6,8 @@
 # stat0 - vollyball-like
 # stat1 - sombrero-like
 # item0 - Item Drop
+# item1 - Booze Drop
+# item2 - Food Drop
```

Comment:
need to correct these post-candy

also rather than (booze, candy, food), (food, booze, candy) might be a more intuitive order?

Reply:
Booze Candy Food is alphabetical ;)

Reply:
yeah but if something new gets added between candy and food you'd have to renumber. Alphabetical isn't so good on lists that could change.

Reply:
Legit. Ok Food, Booze, Candy it is